.github/workflows/tests.yaml

```diff
@@ -15,609 +15,608 @@
 name: Tests
 
 on:
-    push:
-    pull_request:
-    merge_group:
-    workflow_dispatch:
+  push:
+  pull_request:
+  merge_group:
+  workflow_dispatch:
 
 concurrency:
-    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name ==
-        'pull_request' && github.event.number) || (github.event_name ==
-        'workflow_dispatch' && github.run_number) || github.sha }}
-    cancel-in-progress: true
+  group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name ==
+    'pull_request' && github.event.number) || (github.event_name ==
+    'workflow_dispatch' && github.run_number) || github.sha }}
+  cancel-in-progress: true
 
 env:
-    CHIP_NO_LOG_TIMESTAMPS: true
-    # XXX: Workaround for https://github.com/actions/cache/issues/1141
-    SEGMENT_DOWNLOAD_TIMEOUT_MINS: 3
+  CHIP_NO_LOG_TIMESTAMPS: true
+  # XXX: Workaround for https://github.com/actions/cache/issues/1141
+  SEGMENT_DOWNLOAD_TIMEOUT_MINS: 3
 
 jobs:
-    test_suites_linux:
-        name: Test Suites - Linux
-        timeout-minutes: 180
-
-        strategy:
-            matrix:
-                build_variant: [no-ble-tsan-clang]
-                chip_tool: [""]
-        env:
-            BUILD_VARIANT: ${{matrix.build_variant}}
-            CHIP_TOOL_VARIANT: ${{matrix.chip_tool}}
```

Comment:
why such a large diff on this file @rbultman

Reply:
This looks like some sort of over-eager auto-formatter.  The only actual non-whitespace change is the addition of line 143....  Please undo all the whitespace noise.